ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -125,8 +125,20 @@ function is_openshift_version_gte() {
     printf '%s\n%s' "$1" "${DS_OPENSHIFT_VERSION}" | sort -C -V
 }
 
+function extract_oc_for_upgrade() {
+    extract_dir=$(mktemp --tmpdir -d "installer-upgrade-XXXXXXXXXX")
+    _tmpfiles="$_tmpfiles $extract_dir"
```

Comment:
CI is having a problem with this
`environment: line 132: _tmpfiles: unbound variable
`

Reply:
yeah, I'm trying to understand what to do in this case

Reply:
https://github.com/search?q=repo%3Aopenshift-metal3%2Fdev-scripts%20_tmpfiles&type=code 
Seems to be correct based on other codes